README.md

```diff
@@ -6,7 +6,7 @@ Get time passed since you've started node.js process.
 
 ```javascript
 // 1. Set global.appStartTime before you load any node.js modules.
-global.appStartTime = Date.now()
+require('since-app-start').start()
```

Comment:
8-10ms in Brunch on my machine. I would say it is negligible.
Otherwise, we would complicate the API: `global[Symbol.for('since-app-start') = Date.now()`